doc/source/parameter.rst

```diff
@@ -0,0 +1,90 @@
+=============
+``/parameter``
+=============
+
+.. http:get:: /parameter/<parameter_id>
+
+    Get data about the ``<parameter_id>`` parameter
+
+    :>json description: Parameter description.
+    :>json id: Parameter id.
+    :>json values: Parameter values history. Only present if the parameter is a simple one. A ``null`` value means that the parameter does not exist anymore.
+    :>json brackets: Marginal scale history. Only present if the parameter is a marginal scale. A ``null`` value means that the scale does not exist anymore.
```

Comment:
> A ``null`` value means that the parameter does not exist anymore.

This is misleading: `values` can never be null.

Also, I see nowhere documented the types of these properties: `description` and `id` are `String`, while `values` and `brackets` are `Object`…